client/wfnews-war/src/main/angular/src/app/components/report-of-fire/compass-page/rof-compass-page.component.ts

```diff
@@ -98,16 +98,23 @@ handler(e) {
 
     this.reportOfFire.compassHeading = compassHeading;
 
+    this.useMyCurrentLocation();
 }
 
 async useMyCurrentLocation() {
 
   const location = await this.commonUtilityService.getCurrentLocationPromise()
-  if (location) {
+  if (location) {   
     this.currentLat = Number(location.coords.latitude);
     this.currentLong = Number(location.coords.longitude);
   }
+
+  document.getElementById("location").innerText = this.currentLat + "," + this.currentLong;
```

Comment:
Always good to wrap getElement calls by a null check, in case the element doesn't exist. Even if you know it will... sometimes weird bugs pop up this way

Reply:
Thanks I updated that now!